src/kOS/SharedObjects.cs

```diff
@@ -12,15 +12,27 @@ public class SharedObjects : Safe.SafeSharedObjects
         public ProcessorManager ProcessorMgr { get; set; }
         public Part KSPPart { get; set; }
         public TermWindow Window { get; set; }
+        public List<KOSManagedWindow> ManagedWindows { get; }
```

Comment:
Ideally yes, but I didn't want to tread on the code that references SharedObjects.Window. That change should be an Issue.